changelogs/fragments/811-proxmox-kvm-state-absent.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - proxmox_kvm - defer error-checking for non-existent VMs in order to fix idempotency of tasks using ``state=absent`` and properly recognize a success (https://github.com/ansible-collections/community.general/pull/811).
```

Comment:
I don't think this file is needed as it belongs to another PR.

Reply:
Yes, I just started this branch on top of the one used for PR #811. I will rebase if it gets merged.